isic_challenge_scoring/__init__.py

```diff
@@ -59,12 +59,16 @@ def extractZip(zipPath, outputPath, flatten=True):
         raise ScoreException(f'Could not read ZIP file "{zipPath.name}": {str(e)}.')
 
 
-def unzipAll(inputPath):
+def unzipAll(inputPath, allowAbstractDirectory=False):
```

Comment:
Can we rename `allowAbstractDirectory` to use the term "manuscript" (even though the directory is still named "Abstract" for now), just for variable constancy.